### DISCUSSION TITLE
Stripe payment info not updated

### DISCUSSION THREAD

**Discussion by meiremans** [alfio-event/alf.io#1343] [Q&A]
Hi!

When paying through Stripe, it will be stuck on this screen until you press force check.
![image](https://github.com/alfio-event/alf.io/assets/3840355/ce3895e2-7f29-4fde-9cb6-575fa1d11872)

Is there a solution to this?

**Comment by cbellone** [comment]
Hi,
please double check your webhook configuration. 
See [relevant tutorial](https://alf.io/docs/configuration/payment/stripe-sca/)

If everything looks OK, check on [Stripe dashboard](https://support.stripe.com/questions/troubleshooting-webhook-delivery-issues) if there is any problems with the delivery

**Reply by meiremans** [reply]
Hello!

Thank you for your reply.
Everything seems fine. I tried on Stripe API version 2019 and 2023 but I get this in the stripe dashboard.
![image](https://github.com/alfio-event/alf.io/assets/3840355/1fa7fa3b-0296-476a-9d79-ccd771c655ba)

**Reply by cbellone** [reply]
check on the application log. There should be an error. 
You can search the following text: `got exception while handling stripe webhook`
it is printed just before the error

in any case, it's highly likely that you have a wrong configuration somewhere. 
Check https://github.com/alfio-event/alf.io/issues/1254#issuecomment-1677217269 for some pointers
